docs/protocol/extension_parameters.md

```diff
@@ -98,6 +98,12 @@ value. For example to forward all the headers that start with 'PREFIX_' from
 both HTTP and GRPC, you should add `--http-header-forward-pattern PREFIX_.*
 --grpc-header-forward-pattern PREFIX_.*` to your `tritonserver` command.
 
+By default, the regular expression pattern matches headers with case-insensitive
+mode according to the HTTP protocol. If you want to enforce case-sensitive mode,
+simplying adding the `(?-i)` prefix which turns off case-insensitive mode, e.g.
+`--http-header-forward-pattern (?-i)PREFIX_.*`. Note `(?-i)` prefix is not supported
+on HTTP Python client.
```

Comment:
```suggestion
`--http-header-forward-pattern (?-i)PREFIX_.*`. Note, headers sent through the
Python HTTP client may be automatically lower-cased by internal client libraries.
```

Reply:
Updated.

Reply:
Where is the update? I don't see it

Reply:
Bruh. I did not include this change in the commit. Should be there now.